### ISSUE TITLE
build-complete may delete /tmp

### ISSUE THREAD

**Issue by qdii** [OpenVPN/openvpn-build#127]
https://github.com/OpenVPN/openvpn-build/blob/be29cd5c7575da438783bd11412c04f32dfde28c/windows-nsis/build-complete#L10

A [common value](https://en.wikipedia.org/wiki/TMPDIR) for TMPDIR is `/tmp`. If this variable is set, then build-complete will remove everything in it.

**Comment by mattock** [comment]
Good catch. We should definitely document this and maybe even verify that TMPDIR is _not_ set to /tmp. Care to provide a PR?

**Comment by chipitsine** [comment]
I recall erasing my home directory when pointing TMPDIR to it

**Comment by chipitsine** [comment]
I guess the best approach would be

1. create tmp dir from scratch
2. stop build if TMPDIR already exists

**Comment by mattock** [comment]
@chipitsine wouldn't that mean that user would have to wipe TMPDIR manually before every build?  

Perhaps, instead of wiping the TMPDIR entirely in build-complete, we could only wipe the build directories _inside_ it. So, even if TMPDIR was /home/<user> or /tmp there would be no/minimal damage.

**Comment by chipitsine** [comment]
I did have a backup of home dir, so I did not worry much :)
